features/schema.py

```diff
@@ -385,6 +381,9 @@ def resolve_details(self: models.Feature, info, **kwargs):
         for detail in self.details.all():
             # Default dict resolver will resolve this for FeatureDetails
             details[detail.type.lower()] = detail
+
+        # PriceTags have a relation to Feature model, so we resolve it separately
+        details["price_list"] = self.price_tags.all()
```

Comment:
what's the reason for putting `PriceTag`s in `FeatureDetails`?

Reply:
I think that was the intended end goal in PR #63. Some refactoring was done on that PR which left the implementation in a bit unpolished state. :sweat_smile:

Reply:
ok, fair enough. it mentions some slack discussion as the reason for this choice 😄 we'll honor the discussion